### PR TITLE
MAGECLOUD-4105:  Add coverage of new functionality to Functional Testing Framework

### PR DESCRIPTION
**Pull request by oshmyheliuk**
### Fixed Issues (if relevant)

https://magento2.atlassian.net/browse/MAGECLOUD-4105

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: src/Test/Functional/Acceptance/ScenarioExtensibilityCest.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        parent::_before($I);
+        $I->cloneTemplate();
+        $I->addEceComposerRepo();
+        $I->addEceExtendComposerRepo();
+        $I->uploadToContainer('files/debug_logging/.magento.env.yaml', '/.magento.env.yaml', Docker::BUILD_CONTAINER);
+    }
+
+    /**
+     * @param \CliTester $I
+     * @param \Codeception\Example $data
```
Comment (BaDos) [inline src/Test/Functional/Acceptance/ScenarioExtensibilityCest.php]: Remove this `@param`

File: tests/functional/_data/packages/ece-tools-extend/registration.php [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+\Magento\MagentoCloud\ExtensionRegistrar::register('ece-tools-extend', __DIR__);
```
Comment (shiftedreality) [inline tests/functional/_data/packages/ece-tools-extend/registration.php]: add newline in end

**Review by shiftedreality — DISMISSED** [review]

File: tests/functional/_data/packages/ece-tools-extend/registration.php (hunk 2 above)
Comment (BaDos) [inline tests/functional/_data/packages/ece-tools-extend/registration.php]: Add new line

File: src/Test/Functional/Acceptance/ScenarioExtensibilityCest.php (hunk 1 above)
Comment (oshmyheliuk) [reply src/Test/Functional/Acceptance/ScenarioExtensibilityCest.php]: removed

File: tests/functional/_data/packages/ece-tools-extend/registration.php (hunk 2 above)
Comment (oshmyheliuk) [reply tests/functional/_data/packages/ece-tools-extend/registration.php]: added

**Review by BaDos — APPROVED** [review]

**Review by shiftedreality — APPROVED** [review]

**Comment by oshmyheliuk** [comment]
QA not needed, a functional test was added in this pr

**Pull request merged**